app/main.js

```diff
@@ -371,7 +393,10 @@ function windowIconPath () {
     monochrome: settings.get('useMonochromeTrayIcon'),
     inverted: settings.get('useMonochromeInvertedTrayIcon'),
     darkMode: nativeTheme.shouldUseDarkColors,
-    platform: process.platform
+    platform: process.platform,
```

Comment:
Could we not send real data here, but some empty value? I see `windowIconFileName ()` only uses `inverted`, `darkMode` and `monochrome`. Just to see from `main` that it is not used.

Reply:
Sure. I followed what was before, so pause, platform and all below could empty values. To what value set them ???

Reply:
I think `null` should be fine?